REFERENCE.md

```diff
@@ -436,14 +436,32 @@ Start charging the vehicle.
 
 ---
 
+### `send_destination(self)`
```

Comment:
We have to add the latitude, longitude params to the function signature.

i.e.
```
### `send_destination(self, latitude, longitude)`
```